daemon/src/wallet.rs

```diff
@@ -95,16 +102,33 @@ pub struct Actor<B, DB> {
     sender: watch::Sender<Option<WalletInfo>>,
 }
 
+/// Compare the hash of the genesis block of the electrum RPC endpoint to the expected network's
+/// genesis block hash. If they differ, the electrum RPC is not for the network that we expect.
+fn seed_and_rpc_on_same_network(rpc: &electrum_client::Client, network: Network) -> Result<bool> {
```

Comment:
🔧 So that the arguments match the name of the function:

```suggestion
fn seed_and_rpc_on_same_network(rpc: &electrum_client::Client, ext_priv_key: ExtendedPrivKey) -> Result<bool> {
```

Reply:
🙃 Could we move this towards the bottom of the file? I think it's slightly easier to read files if helper code is hidden away down below 😛

Reply:
Sure